content/learn/fundamental-wellbeing.md

```diff
@@ -1,73 +1,96 @@
 ---
-title: "Fundamental Wellbeing"
-created: 2023-04-21
-authors: 
+title: An Introduction to Dr. Jeffery Martin's Work on Fundamental Wellbeing
+created: 2023-11-01
+authors:
   - rufuspollock
   - jefferymartin
+description: Fundamental Wellbeing is the the experience of persistent, higher states of conscious, also known as awakening, enlightenment and other terms. This page is an introduction to the concept.
```

Comment:
"the the" (grammar checker?)

Anyway would rewrite a bit e.g.

> Fundamental Wellbeing is a broad term for experience of persistent, higher states of conscious, also known as awakening, enlightenment and other terms. Dr Jeffery Martin and his colleagues have been conducting rigorous systematic research for more than 15 years and obtained deep insights and some amazing results. This page provides an introduction to the concept and an overview of the work of Martin and his colleagues.

Reply:
Removed the extra "the". 

Description needs to be shorter otherwise it won't show fully as meta data, so left as is